pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib, fetchurl
+# Optional due to unfree license.
+, faacSupport ? false
+, python3Packages, gtk3, wrapGAppsHook
+, gsettings-desktop-schemas, intltool, xvfb-run
+, gobject-introspection, gst_all_1, fdk-aac-encoder }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "soundconverter";
+  version = "4.0.3";
+
+  src = fetchurl {
+    url = "https://launchpad.net/soundconverter/trunk/${version}/+download/${pname}-${version}.tar.gz";
+    sha256 = "sha256-hzIG/4LD3705erPYvXb7uoRwF9LtKKIKB3jrhpYMsZ0=";
+  };
+
+  buildInputs = [
+    gtk3
+    wrapGAppsHook
```

Comment:
You still need `strictDeps = false` otherwise `GDK_PIXBUF_MODULE_FILE` will not be properly set by the wrapper.

Reply:
Okay, but bo idea what `GDK_PIXBUF_MODULE_FILE` is, what is it necessary for?

Reply:
See https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-gdk-pixbuf-loaders. TLDR: It is needed for SVG icons in the interface to render without librsvg being installed globally.

Reply:
Thanks for explaining.